### DISCUSSION TITLE
系统在恢复 (initramfs) 模式下运行。  超出升级包上限

### DISCUSSION THREAD

**Discussion by gaidehaha** [immortalwrt/immortalwrt#1444] [Q&A]
### Describe the bug

我有两台斐讯k2和k2p 
k2升级23，以上系统就无法刷入＞7mb就无法使用，
k2p可以升级，但是升级后提示系统在恢复 (initramfs) 模式下运行。

是不是升级包有问题的原因，用低版本的21，以下的就可以正常升级使用，

### ImmortalWrt version

23.

### ImmortalWrt release

21.02.7

### ImmortalWrt target/subtarget

21.02.7

### Device

k2，k2p

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by nilknow** [comment]
Same issue.
